src/game_database.py

```diff
@@ -0,0 +1,179 @@
+import os
+import sqlalchemy as sql
+from sqlalchemy import text
+from sqlalchemy import Integer, String, Float,Column
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+from config import config
+import pandas as pd
+import logging
+from tqdm import tqdm
+
+
+def create_schema_models(Base):
+    """
+    Creates ORM models for tables within rds/sql database
+    :param Base: This is the base model to build on top of
+    :return `object`: ORM model classes
+    """
+
+    class Metacritic(Base):
```

Comment:
generally, i think it is a better idea to define classes outside of functions. this keeps functions short, and also ensures that score issues do not occur later. currently, all the classes you have defined are local scope (and they get removed from memory stack when function returns)